src/main/java/com/emc/object/s3/S3Client.java

```diff
@@ -390,6 +390,13 @@
 
     AccessControlList getObjectAcl(GetObjectAclRequest request);
 
+    /**
+     * Extend retention <code>period</code>(seconds) on object <code>key</code> in bucket <code>bucketName</code>. NOTE:
+     * New retention period value can only be increased. That is, it can be the same as the current or greater value.
+     * If the new retention period value is -1, infinite retention applies on that object.
```

Comment:
Looks like we need to add -2 in doc
Let's review this https://confluence.cec.lab.emc.com/display/ECS/ECS+3.6+-+Retention+Enhancement+Test+Plan

Reply:
This completely copied from  section "Extending retention period on objects" in <ECS 3.6.1 Data Access Guide>. I decided not to add -2 because the official document does not put this information.

Reply:
Can you check with the author Vlad for why it is added? If necessary, we shall also update dataaccess guide.

Reply:
Per Dev response, -2 is legacy value for CAS retention standing for no retention. Mentioning here might cause unnecessary confusion, so let's make here consistent to what's published in official document.